node/api/client/renter.go

```diff
@@ -43,6 +44,46 @@ func (c *Client) RenterDownloadFullGet(siaPath, destination string, async bool)
 	return
 }
 
+// RenterClearAllDownloadsPost requests the /renter/downloads/clear resource
+// with no parameters
+func (c *Client) RenterClearAllDownloadsPost() (err error) {
+	values := url.Values{}
+	values.Set("newest", "")
```

Comment:
I think if you want the value to be `""` you don't need to set them at all. Same for the other methods